javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+var books = [
+  {
+    isbn: '18291825102',
+    title: 'Getting Over It',
+    author: 'Bennet Foddy'
+  },
+  {
+    isbn: '18987127127',
+    title: 'Kappa',
+    author: 'Prime'
+  },
+  {
+    isbn: '37612375178',
+    title: 'JSON For Dummies',
+    author: 'Unknown'
+  }
+];
+
+console.log('books:', books);
+console.log('typeof books:', typeof books);
+
+var jsonString = JSON.stringify(books);
+console.log('jsonString:', jsonString);
+console.log('typeof jsonString:', typeof jsonString);
+
+var toConvert = '{"number id":"168000","name":"jimothy"}';
```

Comment:
The name of the property shouldn't be "number id", it should just be an `id`, but its value should be a `number`